tailcfg/tailcfg.go

```diff
@@ -231,6 +231,14 @@ type Node struct {
 	//    "https://tailscale.com/cap/file-sharing"
 	Capabilities []string `json:",omitempty"`
 
+	// UnsignedPeerAPIOnly means that this node is not signed nor subject to TKA
+	// restrictions. However, in exchange for that privilege, it does not get
+	// network access. It can only access this node's peerapi, which may not let
+	// it do anything. It is the tailscaled client's job to double-check the
+	// MapResponse's PacketFilter to verify that its AllowedIPs will not be
+	// accepted by the packet filter.
+	UnsignedPeerAPIOnly bool `json:",omitempty"`
```

Comment:
Do we need to bump capver so control knows its safe to do this, or gonna do the bump all at once for the feature this is dependent on?

Reply:
Integers are cheap so I could do it now I guess, but I was going to wait.

But done.